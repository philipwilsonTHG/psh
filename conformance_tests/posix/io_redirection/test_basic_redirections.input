# Test basic I/O redirections
mkdir -p tmp

# Test output redirection
echo "test output" > tmp/out.txt
cat tmp/out.txt

# Test append
echo "appended" >> tmp/out.txt
cat tmp/out.txt

# Test input redirection
echo "input test" > tmp/input.txt
cat < tmp/input.txt

# Test here string only (here docs have parsing issues)
cat <<< "here string test"

# Cleanup
rm -rf tmp