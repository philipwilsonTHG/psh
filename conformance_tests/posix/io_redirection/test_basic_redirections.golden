test output
test output
appended
input test
here string test
